Xamarin.Essentials/Xamarin.Essentials.csproj

```diff
@@ -74,7 +74,7 @@
   <ItemGroup Condition=" $(TargetFramework.StartsWith('MonoAndroid')) ">
     <Compile Include="**\*.android.cs" />
     <Compile Include="**\*.android.*.cs" />
-    <PackageReference Include="System.Numerics.Vectors" Version="4.5.0" />    
+    <PackageReference Include="System.Numerics.Vectors" Version="4.5.0" />
```

Comment:
@webwarrior-ws same here

Reply:
Is preserving 4 trailing spaces **so** important?

Reply:
yes, to prevent git-blame noise, and allow easier cherry-picking or forward-porting later to newer versions (as it would prevent needless possible conflicts).

Reply:
@webwarrior-ws to prevent from committing this kind of hunks again, I recommend always using `git gui` and staging the hunks that you want. DO NOT DO `git add .` recursively blindly, it's a really bad practice.